src/XamlTestCases/XamlTests.cs

```diff
@@ -174,5 +174,29 @@ public void XamlWorkflowWithInputsOutputs()
             Assert.True(outputs.ContainsKey("myOutput"));
             Assert.Equal(1, (int)outputs["myOutput"]);
         }
+
+        [Fact]
+        public void XamlWorkflowWithInputObject()
+        {
+            var xamlString = @"
+                    <Activity x:Class=""WFTemplate""
+                              xmlns=""http://schemas.microsoft.com/netfx/2009/xaml/activities""
+                              xmlns:s=""clr-namespace:System;assembly=mscorlib""
+                              xmlns:s1=""clr-namespace:System;assembly=System""
+                              xmlns:sa=""clr-namespace:CoreWf;assembly=CoreWf""
```

Comment:
Does it work without this line? Most things were renamed to System.Activities.

Reply:
You are right, I started with it before you renamed and did copy + paste of the old one to get started. Works with "System.Activities"